wedlocks/src/__tests__/encrypter.test.js

```diff
@@ -1,26 +1,41 @@
 import forge from 'node-forge'
 import { signParam } from '../encrypter'
+import { vi, beforeAll, afterAll } from 'vitest'
 
-let privateKey
-let publicKey
+beforeAll(() => {
+  vi.useFakeTimers()
+})
 
-// These tests are slow because we generate private keys
-jest.setTimeout(15000)
+afterAll(() => {
+  vi.clearAllTimers()
+})
```

Comment:
Not sure we need this as well as below declaration.  I don't see any use of fake timer in the test. Could you please explain?

Reply:
you are right, not needed!